app/controllers/runtime/info_controller.rb

```diff
@@ -4,6 +4,26 @@ class InfoController < RestController::BaseController
 
     get '/v2/info', :read
     def read
+      unless @config.get(:temporary_enable_v2)
+        return Oj.dump({
+                         name: @config.get(:info, :name),
```

Comment:
Can't you use the "info" map from below and change the "support" and "api_version" values?

Reply:
Good idea